packages/shared/src/components/auth/RegistrationForm.tsx

```diff
@@ -96,7 +100,11 @@ export const RegistrationForm = ({
       formRef?.current ?? form,
     );
 
-    if (!values['traits.name']?.length || !values['traits.username']?.length) {
+    if (
+      !values['traits.name']?.length ||
+      !values['traits.username']?.length ||
+      (showExperienceLevel && !values['traits.experienceLevel']?.length)
```

Comment:
Did you communicate this with Viktor to allow it from Heimdall side?

Reply:
As far as I understand, Heimdall has nothing to do with this, since we just set this data in the database with a graphql mutation after heimdall verifies that the email is eligible for signup. But correct me if I'm wrong :)

Reply:
New `traits` need to be added on Heimdall side to be passed over https://github.com/dailydotdev/heimdall/pull/71

Reply:
Yeah it's something Viktor can help with, he owns the system there.

Reply:
Heimdall PR has been merged now:
- https://github.com/dailydotdev/heimdall/pull/94